about.html

```diff
@@ -11,58 +11,95 @@
 
 </head>
 
+<header>
+    <nav>
+        <div class="navigation">
+       <img class= "logo" src="img/lambda-black.png" alt="">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="products.html">Products</a>
+        <a href="">Blog</a>
+        <a href="contact.html">Contact</a>
+    </div>
+    </nav>
+</header>
+
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        <img src="img/jumbo-about.png" alt="">
+<div class="about">
+    <h4>About</h4>
+      <p>    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        </p>
+      </div>
 
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+    <div class="strategy-collab">
+        <div class="square">
 
+        <img src="img/about-plan.png" alt="strategy">
 
```

Comment:
```suggestion
     <img src="img/about-plan.png" alt="strategy">
```

So alt properties on images are used for accessbility, and should be used to describe exactly what the image is a picture of.

Reply:
You can also use <img src="" _title=""/_>

title will display text if the image is broken, describing what the image is, and it also helps with SEO optimization.